executor/executor_test.go

```diff
@@ -1951,58 +1953,79 @@ func (s *testSuite) TestIssue4024(c *C) {
 	tk.MustQuery("select * from test2.t").Check(testkit.Rows("2"))
 }
 
+const (
+	checkRequestOff          = 0
+	checkRequestPriority     = 1
+	checkRequestNotFillCache = 2
+)
+
 type checkRequestClient struct {
 	tikv.Client
-	priority pb.CommandPri
-	mu       struct {
+	priority     pb.CommandPri
+	notFillCache bool
+	mu           struct {
 		sync.RWMutex
-		turnOn bool
+		checkFlags uint32
```

Comment:
Can't check both priority and notFillCache?

Reply:
It's two separate test, in one testSuite.